packages/main/src/TableCellBase.ts

```diff
@@ -31,6 +32,14 @@ abstract class TableCellBase extends UI5Element {
 	@property({ type: Boolean })
 	_popin = false;
 
+	/**
+	 * Determines the horizontal alignment of table cells.
+	 * Note: All values valid for justify-content can be used not just the ones inside the enum.
+	 * @public
+	 */
+	@property()
+    hAlign?: `${TableCellHorizontalAlign}`;
```

Comment:
hAlign sounds a bit technical, it would make sense to align this with existing property names
Popover: horizontalAlign
MediaGallery: menuHorizontalAlign

Reply:
Done. I've change hAlign to horizontalAlign.